src/org/usfirst/frc/team4571/robot/RobotConstants.java

```diff
@@ -19,4 +19,8 @@
 	public static final double DRIVE_TRAIN_PULSES_PER_REVOLUTION = 1440.0;
 	public static final double DRIVE_TRAIN_WHEEL_CIRCUMFERENCE = DRIVE_TRAIN_WHEEL_DIAMETER_IN_FEET * Math.PI;
 	public static final double DRIVE_TRAIN_DISTANCE_PER_PULSE = DRIVE_TRAIN_WHEEL_CIRCUMFERENCE / DRIVE_TRAIN_PULSES_PER_REVOLUTION;
+
+	public static final int LEFT_MILL_CHANNEL = 1;

```

Comment:
LEFT_MILL_CHANNEL does not describe what it is a channel for. Is it a channel for the motor running the mill? in that case we should probably rename it MILL_LEFT_MOTOR_CHANNEL. and same for the right one as well.